packages/ods/src/components/quantity/tests/navigation/ods-quantity.e2e.ts

```diff
@@ -44,7 +44,7 @@ describe('ods-quantity navigation', () => {
 
   describe('navigation', () => {
     it('should navigate between button & input', async() => {
-      await setup('<ods-quantity></ods-quantity>');
+      await setup('<ods-quantity></ods-quantity><button>Dummy to focus out</button>');
```

Comment:
why ?

Reply:
Test was failing everytime locally, probably an issue when tabbing outside of the current document as there are no other target. With this we still test that the behaviour is as expected and avoid this issue